.releaserc

```diff
@@ -2,24 +2,23 @@
   "branch": "master",
   "tagFormat": "v${version}",
   "verifyConditions": [
-    "@semantic-release/npm",
     {
-      "path": "@semantic-release/github",
```

Comment:
This is removing `@semantic-release/github`, but I believe we would still want this condition. Related to `verifyConditions`, including `@semantic-release/github` here ensures the "precense and validity of the authentication [with GitHub]".

Subsequently, when `@semantic-release/github` is used within `publish` below, it ensures a GitHub release is created in the repository.

Note, the `assets.path` configuration currently used with `@semantic-release/github` can likely stay, pointing to `dist`. It controls which files are included in a release's archive files (e.g., see "Assets" on the GitHub release for the [recently published v4](https://github.com/edx/frontend-logging/releases/tag/v4.0.0)).